### DISCUSSION TITLE
Geo restriction

### DISCUSSION THREAD

**Discussion by WorkMender** [vercel/community#6041] [Help]
### Summary

I have successfully set up my website with Vercel and the website is now live. However, some users from certain parts of the world say they are unable to access the website unless they use a VPN. I have checked online for the DNS propagation however it seems to be globally available. Please what is the issue and how could this be resolved?

### Steps to Reproduce

No Steps
![Screenshot from 2024-03-06 19-24-15](https://github.com/vercel/community/assets/156523391/a84a65c1-a17d-4665-b835-6ece5975f538)

**Comment by WorkMender** [comment]
The county to be precise, is Nigeria. However I opened the vercel application in Australia

**Comment by amyegan** [comment]
Hey @WorkMender. What's the domain for the site?

**Reply by WorkMender** [reply]
It is workmender.com

**Reply by WorkMender** [reply]
Any feedback on this?

**Reply by amyegan** [reply]
Thank you for sharing the domain. Your site is not the only one affected. 

We reached out to the ISP and our vendors to get these sites unblocked. We don't have much control over when that will happen. Thank you for your patience while we work with them to get this solved.
